### PR TITLE
Update device-group-mapping.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated Admin requirements to the document.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/1448

**Comment by PRMerger4** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/enrollment/device-group-mapping.md [hunk 1]
```diff
@@ -51,6 +51,8 @@ You can create any device categories you want. For example:
 
 ## How to configure device categories
 
+You need to be a Global Administrator or Intune Administrator to peform the steps.
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/device-group-mapping.md:54]:
```suggestion
You need to be a Global Administrator or Intune Administrator to perform these steps.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger18** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ErikjeMS** [comment]
#sign-off

**Comment by PRMerger5** [comment]
@ErikjeMS: I'm sorry - only the author of this article, @Lenewsad, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@lenewsad Please review and merge this PR. Thanks

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**